<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1120</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1120</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculadora de TDC</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>calculator_22175.ico</normaloff>calculator_22175.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #333;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: #333;
border-radius:10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1101</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(41, 98, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>451</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 87 13pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);
</string>
        </property>
        <property name="text">
         <string>CALCULADORA  DE TEMPO DE CONTRIBUIÇÃO</string>
        </property>
       </widget>
      </widget>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>130</y>
         <width>511</width>
         <height>301</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>440</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(191, 127, 0);
color: rgb(255, 255, 255);
font: 87 12pt &quot;Arial Black&quot;;
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>LIMPAR/LISTA</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>100</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>100</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>100</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>70</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 87 13pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>DATA/INÍCIO</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>100</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_4">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>100</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>100</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>70</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 87 13pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>DATA/SAIDA</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>440</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(191, 127, 0);
color: rgb(255, 255, 255);
font: 87 12pt &quot;Arial Black&quot;;
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>APAGAR</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_4">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>70</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 66, 99);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 87 13pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>170</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>130</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(191, 127, 0);
color: rgb(255, 255, 255);
font: 87 12pt &quot;Arial Black&quot;;
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>Adicionar Tempos</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>340</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(191, 127, 0);
color: rgb(255, 255, 255);
font: 87 12pt &quot;Arial Black&quot;;
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>LIMPAR</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>850</x>
         <y>120</y>
         <width>241</width>
         <height>161</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>calen.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_2">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>200</y>
         <width>131</width>
         <height>131</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(191, 127, 0);
color: rgb(255, 255, 255);
font: 87 12pt &quot;Arial Black&quot;;
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>CALCULAR</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(191, 127, 0);
color: rgb(255, 255, 255);
font: 87 12pt &quot;Arial Black&quot;;
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>LISTA-ADD</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>890</x>
         <y>300</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(191, 127, 0);
color: rgb(255, 255, 255);
font: 87 12pt &quot;Arial Black&quot;;
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>TOTAL DOS ANOS</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>910</x>
         <y>340</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>390</y>
         <width>281</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>490</y>
         <width>371</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 75 9pt &quot;Verdana&quot;;</string>
       </property>
       <property name="text">
        <string>  Desenvolvido por Khufos ,segue lá no canal do youtube </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>480</y>
         <width>711</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Mensagem de error:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(191, 127, 0);
color: rgb(255, 255, 255);
font: 87 12pt &quot;Arial Black&quot;;
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>SAIR</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="bora.qrc"/>
 </resources>
 <connections/>
</ui>
